### PR TITLE
Enable users to zoom on mobile

### LINKED ISSUE #1473: [Bug]: Cannot zoom in on mobile
### Requirements

- [X] Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a single bug? Do not put multiple bugs in one issue.
- [ ] Is this a server side (not related to the UI) issue? Use the [Lemmy back end](https://github.com/LemmyNet/lemmy) repo.

### Summary

As the title describes. This issue is on the latest release cut of 0.18, but not 0.17.4.

### Steps to Reproduce

1. Open https://lemmy.ml on a mobile device
2. Try to zoom in with the zoom in gesture.

### Technical Details

N/A

### Version

0.18.0-rc.6

### Lemmy Instance URL

lemmy.ml

### PR DESCRIPTION
**Pull request by SleeplessOne1917**
Closes #1473. Was unable to test locally as I would need to run this on a mobile device.

### CODE REVIEW

**Comment by alectrocute** [comment]
This seems like a double-edged sword. Accidental zooming is something I run into frequently with 0.17.

File: src/server/utils/create-ssr-html.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ export async function createSsrHtml(
     <!-- Required meta tags -->
     <meta name="Description" content="Lemmy">
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=yes">
```
Comment (alectrocute) [inline src/server/utils/create-ssr-html.tsx:76]: We can also just omit `user-scalable` entirely since it defaults to `yes`.

**Review by dessalines — APPROVED** [review]

File: src/server/utils/create-ssr-html.tsx (hunk 1 above)
Comment (SleeplessOne1917) [reply src/server/utils/create-ssr-html.tsx]: Omitted.

**Review by alectrocute — APPROVED** [review]

**Comment by SleeplessOne1917** [comment]
> This seems like a double-edged sword. Accidental zooming is something I run into frequently with 0.17.

I just saw this comment. You're probably right about not allowing zoom on mobile would mostly be a better experience, but there is one thing the zoom is really needed for: viewing inline images. Currently, it can be hard to get a bigger view of inline images. Perhaps I should open an issue for a UI feature of a display modal for viewing larger versions of images.

**Pull request merged**